app/controllers/api/v1/participants_controller.rb

```diff
@@ -2,8 +2,40 @@ module Api::V1
   class ParticipantsController < ApplicationController
 
     def index
-      render json: Participant.all
+      render json: Participant.where(status: 'accepted')
     end
 
+    def create
+      participant = Participant.create(email: params[:email],
+                           f_name: params[:f_name],
+                           l_name: params[:l_name],
+                           gender: params[:gender],
+                           phone: params[:phone],
+                           zip: params[:zip],
+                           current_weight: params[:current_weight],
+                           height: params[:height])
+      render json: participant
+    end
+
+  def update
+      begin 
```

Comment:
@mdonagh I see here we could extract the handling of the `ActiveRecord::RecordNotFound` exception to the controller class

```
  rescue_from ActiveRecord::RecordNotFound, with: :not_found

  def not_found
    render json: { message: "Not Found" }, status: 404
  end
```

And in the update action only do something like

```
      @participant = Participant.find(params[:id])
      if @participant.update(status: params[:status])
        render json: @participant, status: :ok
      else
        render json: @participant.errors, status: :unprocessable_entity
      end
```